cylc/flow/platforms.py

```diff
@@ -481,3 +481,9 @@ def get_random_platform_for_install_target(install_target):
     """Return a randomly selected platform (dict) for given install target."""
     platforms = get_all_platforms_for_install_target(install_target)
     return random.choice(platforms)
+
+
+def get_localhost_install_target():
+    """Returns the install target of localhost platform"""
+    localhost = get_platform('localhost')
+    return get_install_target_from_platform(localhost)
```

Comment:
I've tried to kill VSCode's line stripping behavior, but it seems to have become more aggressive. Or perhaps I'm creating more new files....
```suggestion
    return get_install_target_from_platform(localhost)

```